book/src/contributing/releasing.md

```diff
@@ -0,0 +1,66 @@
+    d. `cargo publish` will be run
+    e. A more robust equivalent of `git push && git push --tags` will run
+2. [cargo-dist](https://opensource.axo.dev/cargo-dist/) will see the tagged commit and build binaries/archives/installers/announcements:
+    a. [release.yml](https://github.com/Gankra/cargo-mommy/blob/main/.github/workflows/release.yml) will spawn
+    b. binaries and archives (zips/tarballs) will get built for [all supported targets](https://github.com/Gankra/cargo-mommy/blob/0d96506db241003166e32deb22ad0ab0fc52c16c/Cargo.toml#L41)
+    c. [installers will be built](https://github.com/Gankra/cargo-mommy/blob/0d96506db241003166e32deb22ad0ab0fc52c16c/Cargo.toml#L39) that can fetch the archives
+    d. A [Github Release](https://github.com/Gankra/cargo-mommy/releases) will be made, with the release notes and installers/archives
+3. [oranda](https://opensource.axo.dev/oranda/) will see a workflow called "Release" completed, and build and deploy docs
+    a. [web.yml](https://github.com/Gankra/cargo-mommy/blob/main/.github/workflows/web.yml) will spawn
+    b. [oranda will autodetect, retheme, and build](https://opensource.axo.dev/oranda/book/configuration/mdbook.html) the [cargo-mommy mdbook](https://github.com/Gankra/cargo-mommy/tree/main/src)
+    c. oranda will generate [a website with releases, the book, and a platform-autodetecting install widget for the latest release](https://faultlore.com/cargo-mommy/)
+    d. oranda will push the website to [the gh-pages branch](https://github.com/Gankra/cargo-mommy/tree/gh-pages)
+4. Github Pages will trigger
+    a. [pages-build-deployment](https://github.com/Gankra/cargo-mommy/actions/workflows/pages/pages-build-deployment) will spawn
+    b. [the live website will update](https://faultlore.com/cargo-mommy/)
+
+Note that steps 3 and 4 trigger on all pushes to main, so updating docs doesn't require a new release~
+
+
+
+
+## Updating The Release Process
+
+The cargo-release step was created by hand-copying some config values that Gankra likes to use in her projects. There shouldn't be any need to mess with it.
+
+The cargo-dist step was created by locally running `cargo dist init`, selecting some targets/installers in the interactive prompts, and committing the results. It can be updated by running `cargo dist init` again.
+
+The oranda step was created by running `oranda generate ci`, selecting some of the options, and committing the results. A [custom oranda.json](https://github.com/Gankra/cargo-mommy/blob/main/oranda.json) was then added to fix path-prefix with github-pages, and to pick a non-default theme. The CI is setup to always use the "latest" oranda, so it's self-updating and may never need to be touched. Any changes that are desired will probably be changes to oranda.json.
+
+The Github Pages step was doing in the Github settings UI for cargo-mommy. The website it deploys to is Gankra's.
```

Comment:
Not entirely sure if this is the right edit? It seems good to qualify an explicit reference to the repository per se, even if this is easily inferred, but I don't know if the tense is right.
```suggestion
The Github Pages step was done in the Github Settings UI for the cargo-mommy repo. The website it deploys to is Gankra's.
```